### PR TITLE
Fixed log files documentation inconsistencies

### PR DESCRIPTION
**Pull request by mambelli**
Fixed log files documentation inconsistencies

### CODE REVIEW

File: doc/factory/configuration.html [hunk 1]
```diff
@@ -578,43 +578,53 @@ <h3 class="western"><a name="args_global"></a>Global arguments</h3>
...
               <li>
                 ADMIN: This is a special setting only applicable to Factory logs
                 (and not entry logs) that contains admin messages, such as
-                reconfig comments. This will create a new log called
-                factoryadmin.extension.log. Currently, it cannot be combined
-                with other message types.
+                reconfig comments. This will create a new log appending "admin"
+                to the name, e.g. factoryadmin.extension.log. Currently, it
+                cannot be combined Other message type keywords are ignored.
+                INFO, DEBUG, ERR, and WARN massages are added to the log.
```
Comment (namrathaurs) [inline doc/factory/configuration.html:623]: The new version seems to have removed the second part of the last statement. Is that intended?

File: doc/frontend/configuration.html [hunk 2]
```diff
@@ -517,37 +517,46 @@ <h2 class="western"><a name="management"></a>Frontend Configuration</h2>
...
               >&quot; backup_count=&quot;<i>backup count</i>&quot;
-              type=&quot;<i>ALL</i>&quot; compression=&quot;<i>gz</i>&quot;/&gt;
+              msg_types=&quot;<i>ALL</i>&quot;
+              compression=&quot;<i>gz</i>&quot;/&gt;
             </div>
             <p>
               The admin can configure one or more logs with any combination of
-              following log message types:
+              following log message types. Types are converted to uppercase,
+              unknown typer are silently ignored, the default is INFO,ERR,WARN:
```
Comment (namrathaurs) [inline doc/frontend/configuration.html:534]: typo: "types"

File: CHANGELOG.md [hunk 3]
```diff
@@ -20,6 +20,8 @@ Changes since the last release
 
 ### Bug Fixes
 
+-   Fixed early truncation in log files configuration and inconsistent documentation (PR #462)
```
[the remark this answers is not in the document]
Comment (mambelli) [reply CHANGELOG.md]: done

File: doc/factory/configuration.html [hunk 4]
```diff
@@ -578,43 +578,53 @@ <h3 class="western"><a name="args_global"></a>Global arguments</h3>
...
-              type=&quot;<i>INFO</i>&quot; backup_count=&quot;<i>backup count</i
+              msg_types=&quot;<i>INFO</i>&quot; backup_count=&quot;<i
+                >backup count</i
               >&quot; compression=&quot;<i>gz</i>&quot;/&gt;
             </div>
             <p>
               The admin can configure one or more logs with any combination of
-              following log message types:
+              following log message types. Types are converted to uppercase,
+              unknown typer are silently ignored, the default is INFO,ERR,WARN:
```
[the remark this answers is not in the document]
Comment (mambelli) [reply doc/factory/configuration.html]: done

File: doc/frontend/configuration.html (hunk 2 above)
Comment (mambelli) [reply doc/frontend/configuration.html]: done

**Comment by mambelli** [comment]
This fixes Issue #464

**Review by namrathaurs — APPROVED** [review]
Changes look good!

**Pull request merged**